src/document.js

```diff
@@ -0,0 +1,1135 @@
+/* eslint-disable no-redeclare, no-undef */
+/*
+#     CloudBoost - Core Engine that powers Bakend as a Service
+#     (c) 2014 HackerBay, Inc.
+#     CloudBoost may be freely distributed under the Apache 2 License
+*/
+
+/* eslint no-use-before-define: 0, no-param-reassign: 0 */
+
+const q = require('q');
+const _ = require('underscore');
+const Grid = require('gridfs-stream');
+const winston = require('winston');
+
+const mongodb = require('mongodb');
+const connector = require('./connector');
+const util = require('./util')
+
+const document = {
+
+  get(appId, collectionName, documentId, accessList, isMasterKey) { // returns the document that matches the _id with the documentId
```

Comment:
rename to findById

Reply:
Make this params into an object instead of having multiple params. 
get(appId, collectionName, documentId, accessList, isMasterKey) -> 
findById({ appId, collectionName, documentId, accessList, isMasterKey })